R-package/R/lr_scheduler.R

```diff
@@ -0,0 +1,18 @@
+
+#' @export
+FactorScheduler <- function(step, factor_val) {
+  function(optimizerEnv){
+    num_update <- optimizerEnv$num_update
+    count      <- optimizerEnv$count
+    lr         <- optimizerEnv$lr
+
+    if (num_update > count + step){
+      count = count + step
+      lr = lr * factor_val
+      print(paste("Update[", num_update, "]: learning rate is changed to", lr, "\n"))
```

Comment:
Change this line into `cat(paste("Update[", num_update, "]: learning rate is changed to ", lr, "\n", sep = ""))`

then I will merge it.